### DISCUSSION TITLE
Spacing between lines in the drop-down menu

### DISCUSSION THREAD

**Discussion by arohl** [swiftbar/SwiftBar#298] [Q&A] [answered]
Hi

I am writing a plugin that displays some tabular information and am using PrettyTable in python to generate the table. I am using the style SINGLE_BORDER and when I display the output in the terminal it looks great. However one I display it in SwiftBar, it doesn't look so good as there is space between the lines in the menu - is it possible to change the spacing to be 1.0?

i.e. make

<img width="578" alt="image" src="https://user-images.githubusercontent.com/10984236/159721437-48bd8339-a769-4e0e-8ba9-d5b0bce10a37.png">

look like 
<img width="630" alt="image" src="https://user-images.githubusercontent.com/10984236/159721839-60f36444-7308-478e-aad4-7cd1ece1dbdb.png">

**Comment by melonamin** [comment, marked as answer]
Can you please give me the text version of this? I’ll use it for debug purposes

**Reply by arohl** [reply]
```
┌─────────────────────┬───────────┬──────┬───────┬──────┬────┬────┬───────┐
│ batter              │ dismissal │ runs │ balls │ mins │ 6s │ 4s │   S/R │
├─────────────────────┼───────────┼──────┼───────┼──────┼────┼────┼───────┤
│ Abdullah Shafique   │    caught │   96 │   305 │  465 │  1 │  6 │ 31.47 │
│ Imam-ul-Haq         │       lbw │    1 │    18 │   26 │  0 │  0 │  5.55 │
│ Azhar Ali           │       lbw │    6 │    54 │   63 │  0 │  0 │ 11.11 │
│ Babar Azam          │    caught │  196 │   425 │  603 │  1 │ 21 │ 46.11 │
│ Fawad Alam          │    caught │    9 │    27 │   52 │  0 │  0 │ 33.33 │
│ Mohammad Rizwan     │   not out │  104 │   177 │  235 │  1 │ 11 │ 58.75 │
│ Faheem Ashraf       │    caught │    0 │     1 │    3 │  0 │  0 │  0.00 │
│ Sajid Khan          │    caught │    9 │    10 │   21 │  0 │  2 │ 90.00 │
│ Nauman Ali          │   not out │    0 │    18 │   36 │  0 │  0 │  0.00 │
│ Hasan Ali           │           │      │       │      │    │    │       │
│ Shaheen Shah Afridi │           │      │       │      │    │    │       │
└─────────────────────┴───────────┴──────┴───────┴──────┴────┴────┴───────┘
```

**Reply by melonamin** [reply]
Thnx.

On a first try I couldn't find a way to do it, may be there is a private API I'm not aware of... Not sure I'll be able to achieve this.

Probably not what you're looking for, but another option to show a tabular content could be an "HTML Plugin" using web popover #261

**Reply by arohl** [reply]
Thanks for taking a look! I have been looking at the HTML plugin option and I think it will work well. Only have one question - I can only accomplish this by writing the html to a file and then using a file:: URL?

**Reply by melonamin** [reply]
Yep, exactly, I have an example in this [comment](https://github.com/swiftbar/SwiftBar/issues/261#issuecomment-1013920821)
